raft/log.go

```diff
@@ -95,6 +95,9 @@ func (l *raftLog) maybeAppend(index, logTerm, committed uint64, ents ...pb.Entry
 			l.logger.Panicf("entry %d conflict with committed entry [committed(%d)]", ci, l.committed)
 		default:
 			offset := index + 1
+			if ci-offset>uint64(len(ents)) {
```

Comment:
Suggesting a space around `>` .. it may also fail static check.

Reply:
"`gofmt -w filename`" can format the source code automatically. But I wonder why the CI did not raise an error on this. We may need to enhance the CI.  Just raised an [issue](https://github.com/etcd-io/etcd/issues/13604) to track this. 

@AdamKorcz, could you please point me to the fuzz test case which discovers this issue? I am thinking it might not possible to see this issue in an e2e test case.

Reply:
It did fail with this error https://github.com/etcd-io/etcd/runs/4828699828?check_suite_focus=true#step:5:137

Reply:
@AdamKorcz once you fix the space related static error, we should be good..some tests are flaky and not related to changes. Thanks!